ddd-koans/Introduction/About course/task.html

```diff
@@ -0,0 +1,20 @@
+<html>
+<h3>
+    <b>Welcome to the DDD Koans course!</b>
+    <br/>
+    <br/>
+    You will learn how to apply Domain-Driven Design tactical building blocks using Kotlin.
+    <br/>
+    <br/>
+    In this course, we would like to focus on the domain of music.
+    <br/>
+    <br/>
+    No worries, you don't need to have an understanding of music theory.
+    <br/>
```

Comment:
Why is this the only place where is only one <br> markup?

Reply:
just visually it looked better for me

Reply:
done